lib/utils/deploy.go

```diff
@@ -0,0 +1,144 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+
+	g "github.com/sdslabs/katana/configs"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func DeployChallenge(chall_name, team_name string, patched bool) error {
+
+	team_namespace := team_name + "-ns"
+
+	log.Println(team_namespace)
+	kubeclient, err := GetClient(g.KatanaConfig.KubeConfig)
+	if err != nil {
+		return err
+	}
+
+	deploymentsClient := kubeclient.AppsV1().Deployments(team_namespace)
+	imageName := chall_name + ":latest"
+	if patched {
```

Comment:
`patched` should be something like `firstPatch` or something like this, it is difficult to understand from the variable name currently, bit ambiguous.